### ISSUE TITLE
How to use with full_input_video?

### ISSUE THREAD

**Issue by JasonS09** [kijai/ComfyUI-EasyAnimateWrapper#7]
I noticed the EasyAnimate Sampler includes a full_input_video input. However, no matter what I try, I get this error if I pass a video as input.

`RuntimeError: The size of tensor a (7) must match the size of tensor b (6) at non-singleton dimension 2`

**Comment by kijai** [comment]
It's meant for video2video, I'm not force resizing that input so you probably want to use this node to resize to closest compatible dimensions: 

![image](https://github.com/user-attachments/assets/91d4a27d-8c84-422b-8cd6-0ff675ed6f26)

Also if you didn't try inputting only the video without start or end images. I don't recognize that error though, so I'm unsure of the cause.

**Comment by JasonS09** [comment]
I still get the error, here's my workflow
![Captura de pantalla 2024-08-11 182903](https://github.com/user-attachments/assets/4966e204-d656-42dc-9465-960e6395ecbb)

Also, are the image inputs (image_start, image_end, full_input_video) meant to be exclusive with each other? Or can I use all the inputs simultaneously?

**Comment by kijai** [comment]
Using them with full video would only replace the first or last frame, so not very useful. Have you tried different frame counts? It might need even number or something.

**Comment by JasonS09** [comment]
I just tried with 24, I still get the same error but the numbers change:

`RuntimeError: The expanded size of the tensor (220) must match the existing size (348) at non-singleton dimension 3.  Target sizes: [2, 16, 19296, 220].  Tensor sizes: [2, 16, 1, 348]`

**Comment by kijai** [comment]
Yeah now I'm getting that too, maybe Comfy changed something with the text encoder since this definitely worked before.

**Comment by kijai** [comment]
Nevermind sorry, it's because the image encoder can only take single image, not the whole video.

**Comment by kijai** [comment]
https://github.com/user-attachments/assets/a39dedbb-00fa-41e9-895e-5b016f7ee57e

**Comment by JasonS09** [comment]
Hello, tried to run vid2vid again using the example, this time it didn't work and got the same error. Can you reproduce it?
